### ISSUE TITLE
Default <button> cursor should be configurable via _settings.scss

### ISSUE THREAD

**Issue by brentdodell** [zurb/foundation-sites#10366]
Should [default `cursor`](https://github.com/zurb/foundation-sites/blob/develop/scss/_global.scss#L205-L217) be a pointer (or a value configurable by `_settings.scss`), indicating that the button is clickable?

#### How to reproduce this bug:
Hover over a `<button>` element.

#### What should happen:
Default `cursor` should be a pointer (or a value configurable by `_settings.scss`), indicating that the button is clickable.

#### What happened instead:
`cursor` CSS is not set, defaulting to `auto` and displaying an arrow.

#### Browser(s) and Device(s) tested on:
Chrome on OSX

#### Foundation Version(s) you are using:
6.4.1, XY Grid

#### Test case link:
https://codepen.io/brentdodell/pen/WOgaov

**Comment by IamManchanda** [comment]
> Default cursor should be a pointer (or a value configurable by _settings.scss), indicating that the button is clickable.

:+1:

**Comment by IamManchanda** [comment]
Thanks ... Added a safe PR #10367

**Comment by brettsmason** [comment]
Not sure if this is advisable or not - a quick search seems to suggest a button shouldn't have a pointer, eg: https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

**Comment by IamManchanda** [comment]
@brettsmason I didnt add `pointer` ... i just binded auto into a variable if anyone wants to change it! 
That being said ... i think we can consider `cursor: default` too

**Comment by brettsmason** [comment]
@IamManchanda Yeah no problems just wanted to flag up that its generally not best practice to set to pointer that's all, and that was the original question 😄

**Comment by IamManchanda** [comment]
@brettsmason `default` or `auto` ??

**Comment by brettsmason** [comment]
@IamManchanda Looks like `auto` is the default, so I'd go with that.

**Comment by brentdodell** [comment]
Thank you both @IamManchanda & @brettsmason.

I understand that it may not a best practice, but it seems many people do it ([Foundation buttons even do it](https://github.com/zurb/foundation-sites/blob/v6.4.1/scss/components/_button.scss#L83-L111)) and I do think it would be useful to have it in the settings.

Also, @brettsmason I do not say this to contradict you, but I found it humorous that the article you linked to had a button at the bottom of the page that was styled with `cursor: pointer;`. 

<img width="1440" alt="screen shot 2017-07-07 at 1 52 39 pm" src="https://user-images.githubusercontent.com/9657923/27972524-4db078da-631c-11e7-9373-2315075eae17.png">

Thanks again to you both!

**Comment by brentdodell** [comment]
> @IamManchanda Yeah no problems just wanted to flag up that its generally not best practice to set to pointer that's all, and that was the original question 😄

I wasn't aware of that. Thank you!

**Comment by brettsmason** [comment]
@brentdodell Ha that's great irony right there, thanks for pointing it out 😄 
You are right though, no harm in having it configurable 👍
